langchain/llms/mlflow_gateway.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+from typing import Any, List, Dict, Mapping, Optional
+from langchain.callbacks.manager import CallbackManagerForLLMRun
+from langchain.llms.base import LLM
+from pydantic import BaseModel, Extra
+
+
+class Params(BaseModel, extra=Extra.allow):
+    temperature: float = 0.0
+    stop: List[str] | None = None
+    max_tokens: Optional[int] = None
+    candidate_count: int = 5
```

Comment:
Should we leave this as a default 1? We do have a provider that only supports the value of max '1' and it seems like defaulting to the cheaper of all options is probably something that users would appreciate :)

Reply:
We also only ever return the first candidate from the `_call()` below anyway

Reply:
sounds good!